src/com/cloudogu/ces/dogubuildlib/EcoSystem.groovy

```diff
@@ -146,17 +146,27 @@ class EcoSystem {
         }
     }
 
-    void runYarnIntegrationTests(int timeoutInMinutes, String nodeImage) {
+    void runYarnIntegrationTests(int timeoutInMinutes, String nodeImage, ArrayList<String> additionalArgs=[], boolean enableVideoRecording=false) {
         script.sh 'rm -f integrationTests/it-results.xml'
+        def additionalContainerRunArgs = "${parseAdditionalIntegrationTestArgs(additionalArgs)} "
 
         script.timeout(time: timeoutInMinutes, unit: 'MINUTES') {
             try {
-                script.withZalenium { zaleniumIp ->
-                    script.dir('integrationTests') {
-
-                        script.docker.image(nodeImage).inside("-e WEBDRIVER=remote -e CES_FQDN=${externalIP} -e SELENIUM_BROWSER=chrome -e SELENIUM_REMOTE_URL=http://${zaleniumIp}:4444/wd/hub") {
-                            script.sh 'yarn install'
-                            script.sh 'yarn run ci-test'
+                def defaultConfig = [seleniumVersion      : '3.141.59-p42',
```

Comment:
We don't need to define the complete map with all parameters. The config that is given to `script.withZalenium(defaultConfig, zaleniumNetwork)` is merged into the default configuration of the zalenium lib. With the current implementation, we would need to maintain both. Therefore, it is sufficient to define the map as follows:
```
def defaultConfig = [videoRecordingEnabled: enableVideoRecording]
```